src/test/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptorTest.java

```diff
@@ -467,6 +467,28 @@ public void testDisableDefaultsAndReenableWithDifferentArgs() {
         Assert.assertEquals(rf.maxReadLength.intValue(), 13);
     }
 
+    // TODO this test is enforcing the expected behavior that if a tool declares a default annotation with an optional argument
+    // TODO and if the user then disables and reenables that argument, that the tools default optional argument value will be
+    // TODO used over the global default, which seems wrong. See https://github.com/broadinstitute/gatk/issues/3848
```

Comment:
This is a known issue in barclay (https://github.com/broadinstitute/barclay/issues/23). Maybe it should be mentioned here too (and this disabled test correspond to a different PR, I guess...)

Reply:
I *think* this is a different issue.